tests/NgTests/NgTests.csproj

```diff
@@ -32,233 +32,26 @@
     <ErrorReport>prompt</ErrorReport>
     <WarningLevel>4</WarningLevel>
   </PropertyGroup>
+  <PropertyGroup>
```

Comment:
>PropertyGroup [](start = 3, length = 13)

Can we merge the `PropertyGroup`s with no `Condition` to the main `PropertyGroup`?

Reply:
It looks like most `csproj`s have similar issues. Not a big deal, feel free to ignore.

Reply:
We took this approach in gallery. It's just a way to call out the properties use for pack.